### ISSUE TITLE
automate accessibility review with AccessLint

### ISSUE THREAD

**Issue by ckundo** [rails/rails#40305]
### Steps to reproduce

- Add a Markdown image to the docs, that is missing alt text.
- or, add an input with a missing accessible name, e.g. missing `label`

### Expected behavior

- Expect automated review to alert the author and maintainers to the accessibility issues.

### Actual behavior

- The issue may go unnoticed in review and be deployed to production.

You can add [AccessLint GitHub App](https://github.com/apps/accesslint), it'll execute this kind of code review as illustrated in https://github.com/AccessLint/rails/pull/2. it's free for open source projects.

![Screen Shot 2020-09-29 at 9 57 07 AM](https://user-images.githubusercontent.com/108163/94568607-b603e780-023a-11eb-9d38-69a5aeccf5e5.png)

**Comment by ckundo** [comment]
@seanpdoyle please take a look!

**Comment by ckundo** [comment]
once the app is installed, I can open a test PR to spot check https://github.com/rails/rails/compare/master...AccessLint:accesslint/test

**Comment by composerinteralia** [comment]
Thanks for the suggestion, Cameron, and good to see you! Adding this seems like a good idea to me. I will look into it.

**Comment by composerinteralia** [comment]
I clicked on some things, and I don't think I have the permissions to add it. If somebody who has the right permissions comes across this issue, I have used this on several projects at thoughtbot and it has been helpful.

**Comment by rafaelfranca** [comment]
While I see the value of this, there are few things that I'd like to be different in order to enable. We prefer to have a tool that executes in a github action, which don't flood PRs with comments but still point problems so people can fix.

Is it possible to setup this tooling like this?

**Comment by ckundo** [comment]
@rafaelfranca do you mean an Action that posts to the Issues tab, instead of comments? the advantages of comments approach are that it doesn't break the build for issues that are sometimes subjective, and is also a good socialization and learning. Some accessibility issues require discussion, like what to name an input or image. However, some issues could be pushed to the issues tab, like invalid aria content.

That said, please consider trying AccessLint out as is to see how it feels. I'll take a look at an Action and post a comment with findings. Thanks!

**Comment by ckundo** [comment]
I looked into defining a GitHub Action, and to be honest, there's a lot of work there to port the App. Initial spike is here if you're curious: https://github.com/AccessLint/actions/tree/sandbox.

**Comment by simi** [comment]
@ckundo that link results into 404. Isn't that private repo?

**Comment by ckundo** [comment]
@simi whoops, yep! just switched it to public, you can take a look now..
